src/parameterized_tendencies/microphysics/microphysics_wrappers.jl

```diff
@@ -34,6 +34,30 @@ function limit(q, dt, n::Int)
     return q / float(dt) / n
 end
 
+function ml_N_cloud_liquid_droplets(
+    α_dust,
+    α_seasalt,
+    α_SO4,
+    α_q_liq,
+    c₀_dust,
+    c₀_seasalt,
+    c₀_SO4,
+    N₀,
+    c_dust,
+    c_seasalt,
+    c_SO4,
+    q_liq,
+)
+    # We can also add w, T, RH, w' ...
+    # Also consider lookind only at around cloud base height
+    FT = eltype(N₀)
+    return N₀ +
+           α_dust * log(max(c_dust, eps(FT)) / c₀_dust) +
+           α_seasalt * log(max(c_seasalt, eps(FT)) / c₀_seasalt) +
+           α_SO4 * log(max(c_SO4, eps(FT)) / c₀_SO4) +
+           α_q_liq * log(max(q_liq, eps(FT)))
+end
```

Comment:
```suggestion
function ml_N_cloud_liquid_droplets(
    cmc,
    c_dust,
    c_seasalt,
    c_SO4,
    q_liq,
)
    (; α_dust,
    α_seasalt,
    α_SO4,
    α_q_liq,
    c₀_dust,
    c₀_seasalt,
    c₀_SO4,) = cmc.aml
    N₀ = cmc.N_cloud_liquid_droplets
    # We can also add w, T, RH, w' ...
    # Also consider lookind only at around cloud base height
    FT = eltype(N₀)
    return N₀ +
           α_dust * log(max(c_dust, eps(FT)) / c₀_dust) +
           α_seasalt * log(max(c_seasalt, eps(FT)) / c₀_seasalt) +
           α_SO4 * log(max(c_SO4, eps(FT)) / c₀_SO4) +
           α_q_liq * log(max(q_liq, eps(FT)))
end
```

Reply:
I don't have a good sense if we should use log q_liq here, and if we do, whether we should divide it by a reference q_liq as well. Maybe we can start with this and see what we get. Also, will negative N cause an issue? Do you want to clip it?

Reply:
If N is negative the CloudMicrophysics will return zero for effective radius. Which will be clipped in the radiation lookup table to the smallest available size. So I think we should be safe(ish)